.github/workflows/ci.yml

```diff
@@ -93,7 +93,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18.x
-          cache: "yarn"
+          cache: 'yarn'
```

Comment:
why is that?

Reply:
https://github.com/l2beat/l2beat/pull/4291/commits/42981c6bc52b2a00d5ce9331af589bbb29cd2cce

Leftover - was debugging CI